CHANGELOG.md

```diff
@@ -50,6 +50,7 @@
 - Added support to turn off trailing dots for grafana.
 - Added gatekeeper mutation to enable changing ndots for any specified pod/podgroup or clusterwide.
 - Added templating to configure Falco with different syscall sources (default is still `module`)
+- Extra component versions can be added in the Welcome dashboard via config
```

Comment:
Shouldn't this be in wip-changelog and where are the migration steps that the checklist says you have done?

Reply:
Right. Confused Monday-brain put it in the wrong changelog file, pushing a fixup commit.

Migration steps must have been for a different PR, or I hit the wrong checkbox. This one should not need a migration, will move the checkmark.